tools/packaging/debian/precice.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=/usr
```

Comment:
When doing `sudo make install` in my system, all the files are installed to `/usr/local`, while if I make a package and install it, they go to `/usr/`.

Therefore, this does not work with `make install`.

Reply:
In that case, you have to set the install prefix using
```
cmake -DCMAKE_INSTALL_PREFIX=/usr/ [...]
```

Reply:
Good point, but apparently this is a different issue (see other comment).